Free Ruler/Base.lproj/MainMenu.xib

```diff
@@ -238,11 +246,5 @@
             </items>
             <point key="canvasLocation" x="-147" y="125"/>
         </menu>
-        <menuItem title="Group Rulers" state="on" keyEquivalent="g" id="7SO-Lm-Ylr">
-            <modifierMask key="keyEquivalentModifierMask"/>
-            <connections>
-                <action selector="toggleGroupRulers:" target="Voe-Tx-rLC" id="RkN-cz-hMe"/>
-            </connections>
-        </menuItem>
```

Comment:
this was a weird stray menuitem that somehow got added outside the menu, not visible in the app. A duplicate of the existing Group Rulers menu item.